subsys/bluetooth/services/cts.c

```diff
@@ -7,9 +7,16 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+
+#ifdef CONFIG_BT_CTS_HELPER_API
+#undef _POSIX_C_SOURCE
+#define _POSIX_C_SOURCE 200809L
```

Comment:
Maybe add a code comment here explaining why you're doing this (i.e. which libc API needs this)

Reply:
Done